tests/performance/test_performance.cpp

```diff
@@ -58,6 +58,48 @@ main(int argc, char* argv[]) {
     std::string build_parameters = argv[4];
     std::string search_parameters = argv[5];
 
+    if (build_parameters == "hello hgraph") {
```

Comment:
what's the mean of `hello hgraph`

Reply:
the origin param is too long，so I mark another name to do this, I will reset the changes in test_performance and examples